op-deployer/pkg/deployer/manage/add_game_type_test.go

```diff
@@ -0,0 +1,64 @@
+package manage
+
+import (
+	"context"
+	"log/slog"
+	"math/big"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/ethereum-optimism/optimism/op-deployer/pkg/deployer/opcm"
+	"github.com/ethereum-optimism/optimism/op-deployer/pkg/deployer/testutil"
+	"github.com/ethereum-optimism/optimism/op-service/testlog"
+	"github.com/ethereum-optimism/optimism/op-service/testutils"
+	"github.com/ethereum-optimism/superchain-registry/validation"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/stretchr/testify/require"
+)
+
+func TestAddGameType(t *testing.T) {
+	rpcURL := os.Getenv("SEPOLIA_RPC_URL")
+	require.NotEmpty(t, rpcURL, "must specify RPC url via SEPOLIA_RPC_URL env var")
+
+	afacts, _ := testutil.LocalArtifacts(t)
+	v200SepoliaAddrs := validation.StandardVersionsSepolia["op-contracts/v2.0.0-rc.1"]
+	testCacheDir := testutils.IsolatedTestDirWithAutoCleanup(t)
+
+	cfg := AddGameTypeConfig{
+		L1RPCUrl:         rpcURL,
+		Logger:           testlog.Logger(t, slog.LevelInfo),
+		ArtifactsLocator: afacts,
+		Input: opcm.AddGameTypeInput{
+			SaltMixer: "foo",
+			// The values below were pulled from the Superchain Registry.
+			SystemConfig:            common.HexToAddress("0x034edD2A225f7f429A63E0f1D2084B9E0A93b538"),
+			ProxyAdmin:              common.HexToAddress("0x189aBAAaa82DfC015A588A7dbaD6F13b1D3485Bc"),
+			DelayedWETH:             common.HexToAddress("0x9C7750C1c7b39E6b0eFeec06A1F2cf06190f6018"),
```

Comment:
Could you populate these inputs with values read from the superchain-registry package instead of hardcoding? Even better would be to only ready SystemConfigProxy from that package then use op-fetcher to grab the other addresses from onchain. Also curious if this test will break when op-sepolia upgrades contracts in the future

Reply:
it may break in the future, but this is the best I have for now